kontrakter/politi/varetekt/1.0/begjaeringVaretekt.schema.json

```
@@ -105,33 +113,596 @@
+        "dNummer": {"type": "array", "items": {"$ref": "#/definitions/dNummer"}},
+        "adresseGradering": {"$ref": "#/definitions/adresseGradering"},
+        "personAdresse": {"$ref": "#/definitions/personAdresse"}
+      },
+      "required" : ["internId","navn","foedselsdato", "statsborgerskap", "foedselsnummer", "dNummer"],
+      "additionalProperties": false
+    },
+
+    "personMini": {
+      "description": "Kun navn og fødselsdato (hvis den finnes)",
+      "type": "object",
+      "properties": {
+        "internId": {"type": "string"},
+        "navn": {"$ref" : "#/definitions/personnavn"},
+        "foedselsdato": {"type" : "string", "format" : "date"}
+      },
+      "required" : ["internId","navn"],
+      "additionalProperties": false
+    },
+	  "profesjonellPerson": {
+      "description": "Advokater, tolker, sakkyndige vitner",
+      "type": "object",
+      "properties": {
+        "tittel" :  {"type": "string"},
+        "navn": {
+          "type" : "object",
+          "properties": {
+            "fornavn" : {"type": "string"},
+            "mellomnavn" : {"type": "string"},
+            "etternavn" : {"type": "string"}
+            },
+          "required" : ["etternavn"],
+          "additionalProperties": false
+        },
+        "kontakt" : {"$ref" : "#/definitions/kontaktInfo"}
+      },
+      "required" : ["navn"],
+      "additionalProperties": false
+    },
+
+    "basissak" : {
+      "description": "Person eller foretak og straffesaksreferanse",
+      "type": "object",
+      "properties": {
+        "straffesaksnummer" : {"type" : "string"},
+        "personForetak": {"$ref": "#/definitions/personForetakMini"}
+      },
+      "required" : ["straffesaksnummer", "personForetak"],
+      "additionalProperties": false
+    },
+
+    "grunnlagBasisaker" : {
+      "description": "Grunnlaget er beskrivelse og referanser til basissaker (rolle,sak)",
+      "type": "object",
+      "properties": {
+        "bokstav" : {"type" : "string"},
+        "grunnlagstekst": {"type" : "string"},
+        "basissaker": {"type": "array", "items": {"$ref": "#/definitions/basissak"}, "minContains": 1}
+      },
+      "required" : ["grunnlagstekst","basissaker"],
+      "additionalProperties": false
+    },
+    "punktDetaljer" : {
+      "description": "Lovbudreferanse, grunnlag med basissaker",
+      "type": "object",
+      "properties": {
+        "nummer" : {"type" : "string"},
+        "lovbud": {"type": "array", "items": {"$ref": "#/definitions/lovbudGjengivelse"},"minItems" : 1,"maxItems": 2},
+        "grunnlag": {"type": "array", "items": {"$ref": "#/definitions/grunnlagBasisaker"}}
+      },
+      "required" : ["lovbud"],
+      "additionalProperties": false
+    },
+
+    "punkt" : {
+      "description" : "Et tiltalepunkt med romertall",
+      "type": "object",
+      "properties": {
+        "nummer" : {"type" : "string"},
+        "prinsipalPunkt" :{"$ref": "#/definitions/punktDetaljer"}
```

Er det nødvendig med dette ekstra steget inn i `punktDetaljer`? Kan ikke alt som ligger i `punktDetaljer` bare ligge i `punkt`?
```suggestion
        "lovbud": {"type": "array", "items": {"$ref": "#/definitions/lovbudGjengivelse"},"minItems" : 1,"maxItems": 2},
        "grunnlag": {"type": "array", "items": {"$ref": "#/definitions/grunnlagBasisaker"}} 
```

---

Glemt subsidiært tiltlapunkt som er grunnet til punktDetaljer. Lagt til nå